[E]69.cpp

```diff
@@ -0,0 +1,19 @@
+// Problem: 69. Sqrt(x) - Easy
+// Link: https://leetcode.com/problems/sqrtx/
+
+class Solution {
+public:
+    int mySqrt(int x) {
+        // cout << (int)sqrt(INT_MAX) << endl;
+        // I must cout to find the max of range first, before searching.
+        int l = 0, r = 46340;
+        while (l <= r) {
+            int mid = (l + r) / 2;
+            int p = mid * mid;
+            if (p < x) l = mid + 1;
```

Comment:
Sau if, else nên cho hết vào trong 1 cặp ngoặc {}. Kể cả chỉ là 1 lệnh;
Vd : if (p <x) {
           l = mid  + 1;
        } else if (p > x) {
           r = mid - 1;
        }

Reply:
Dùng cặp ngoặc thì đúng là đẹp hơn nhưng code thường hơi dài. Chú nghĩ vấn đề đó thế nào?

Reply:
Nên tập thói quen cho tất cả trong dấu ngoặc như a nói. Code sẽ đẹp hơn. Các công ty to đều đánh giá cao phong cách code đấy. Họ ko thích code ngắn mà khó đọc thế này.

Reply:
Ok.

Reply:
ok sửa đi nhá cho đẹp.